main.js

```diff
@@ -13,6 +13,18 @@ let getToken = () => {
 
 token = getToken();
 
+let loginButtonStatus = () => {
```

Comment:
We may be able to combine these two functions since they have some repetitive qualities